### ISSUE TITLE
A11y_Visual Studio Code_HomePage_Non-Text Contrast: Previous and Next buttons in home page are not meeting required contrast ratio 3:1

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4511]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#WCAG1.4.11#Non-Text Contrast#Visual Studio Code Services#VisualStudioCode-Web-Apr2021
### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)
### Repro Steps:

1. Hit [Visual Studio Code](https://code.visualstudio.com/) link
2. Navigate to next and previous button on the carousel.
3. Verify that contrast ratio of previous and next buttons with background is 1.2:1 which is less that required contrast ratio 3:1.
### Actual Result:

Contrast ratio of previous and next buttons with background is 1.2:1 which is less that required contrast ratio 3:1. 
### Expected Result:

Contrast ratio of previous and next buttons with background should be greater than or equal to required contrast ratio 3:1. 
### User Impact:
Users with color blindness will not be able to identify the previous and next buttons present on the page if contrast ratio is less than 3:1
### Attachment:
![A11y_Visual Studio Code Client_HomePage_Non-Text Contrast](https://user-images.githubusercontent.com/83570638/116857294-2e3fea80-ac1a-11eb-8ec9-af7a567529ea.png)

**Comment by brandonalanhuffman** [comment]
I need help with my desktop i have had a bug installed and need help
repairing it. I will pay to fix this. I think its a registry issue.

Brandon

**Comment by VijayaBharathyA** [comment]
#Closed;
**Verified issue in Below environment:**
URL: https://code.visualstudio.com/
Issue is Fixed 
![image](https://user-images.githubusercontent.com/100189407/217511981-0bfc62bd-aaa2-4edb-ba69-36429aa6d213.png)
